### PR TITLE
Blog week 39

### PR DESCRIPTION
**Pull request by sakalosj**

### CODE REVIEW

File: content/posts/September-2020.md [hunk 1]
```diff
@@ -46,3 +46,15 @@ draft: false
   You can now use `test` job also on Pagure and GitLab thanks to that.
 
 [laura]: https://github.com/lbarcziova
+
+## Week 39 (September 21st - September 25th)
+
+- Service supports private namespace now. It needs to be configured in the service config. [packit-service#831](https://github.com/packit/packit-service/pull/831)
```
Comment (jpopelka) [inline content/posts/September-2020.md:52]:
```suggestion
- Service supports private namespaces now. They need to be configured in the service config. [packit-service#831](https://github.com/packit/packit-service/pull/831)
```

File: content/posts/September-2020.md [hunk 2]
```diff
@@ -46,3 +46,15 @@ draft: false
...
+- If git tag contains not only version (e.g. `pkg_name-v1.2.3`), it is possible to use upstream_tag_template to extract version from tag, which will be used subsequent task. [doc](https://packit.dev/docs/configuration/#upstream_tag_template) [packit#959](https://github.com/packit/packit/pull/959)
+- Added support for globbing pattern in upstream_ref. [doc](https://packit.dev/docs/configuration/#upstream_ref) [packit#960](https://github.com/packit/packit/pull/960)
+- Packit `--remote` is global option now and available for all commands. Because of this `sync-from-downstream --remote` was renamed to `--remote-to-push`. Remote can now be specified in the user's config (via upstream_git_remote parameter). [packit#977](https://github.com/packit/packit/pull/977)
+
+Following bugs were fixed:
+
+- Packit was dropping leading zeros in version anymore. [packit#814](https://github.com/packit/packit/pull/814)
```
Comment (jpopelka) [inline content/posts/September-2020.md:59]:
```suggestion
- Packit dropping leading zeros in version. [packit#814](https://github.com/packit/packit/pull/814)
```

File: content/posts/September-2020.md [hunk 3]
```diff
@@ -46,3 +46,15 @@ draft: false
   You can now use `test` job also on Pagure and GitLab thanks to that.
 
 [laura]: https://github.com/lbarcziova
+
+## Week 39 (September 21st - September 25th)
+
+- Service supports private namespace now. It needs to be configured in the service config. [packit-service#831](https://github.com/packit/packit-service/pull/831)
+- If git tag contains not only version (e.g. `pkg_name-v1.2.3`), it is possible to use upstream_tag_template to extract version from tag, which will be used subsequent task. [doc](https://packit.dev/docs/configuration/#upstream_tag_template) [packit#959](https://github.com/packit/packit/pull/959)
```
Comment (jpopelka) [inline content/posts/September-2020.md:53]:
```suggestion
- If git tag contains not only version (e.g. `pkg_name-v1.2.3`), it is possible to use upstream_tag_template to extract version from tag, which will be used in a subsequent task. [doc](https://packit.dev/docs/configuration/#upstream_tag_template) [packit#959](https://github.com/packit/packit/pull/959)
```

**Review by jpopelka — APPROVED** [review]

File: content/posts/September-2020.md [hunk 4]
```diff
@@ -46,3 +46,15 @@ draft: false
   You can now use `test` job also on Pagure and GitLab thanks to that.
 
 [laura]: https://github.com/lbarcziova
+
+## Week 39 (September 21st - September 25th)
+
+- Service supports private namespaces now. They need to be configured in the service config. [packit-service#831](https://github.com/packit/packit-service/pull/831)
+- If git tag contains not only version (e.g. `pkg_name-v1.2.3`), it is possible to use upstream_tag_template to extract version from tag, which will be used in a subsequent task. [doc](https://packit.dev/docs/configuration/#upstream_tag_template) [packit#959](https://github.com/packit/packit/pull/959)
```
Comment (TomasTomecek) [inline content/posts/September-2020.md:53]:
```suggestion
- If git tag contains more information than just version (e.g. `pkg_name-v1.2.3`), it is possible to use upstream_tag_template to extract version from the tag, which will be used in a subsequent task. [doc](https://packit.dev/docs/configuration/#upstream_tag_template) [packit#959](https://github.com/packit/packit/pull/959)
```

File: content/posts/September-2020.md [hunk 5]
```diff
@@ -46,3 +46,15 @@ draft: false
   You can now use `test` job also on Pagure and GitLab thanks to that.
 
 [laura]: https://github.com/lbarcziova
+
+## Week 39 (September 21st - September 25th)
+
+- Service supports private namespaces now. They need to be configured in the service config. [packit-service#831](https://github.com/packit/packit-service/pull/831)
```
Comment (TomasTomecek) [inline content/posts/September-2020.md:52]: I'd mention this is not available for github and we need it for CentOS Stream.

File: content/posts/September-2020.md [hunk 6]
```diff
@@ -46,3 +46,15 @@ draft: false
   You can now use `test` job also on Pagure and GitLab thanks to that.
 
 [laura]: https://github.com/lbarcziova
+
+## Week 39 (September 21st - September 25th)
+
+- Service supports private namespaces now. They need to be configured in the service config. This is required in CentOS stream and not avaialble in GitHub. [packit-service#831](https://github.com/packit/packit-service/pull/831)
```
Comment (lachmanfrantisek) [inline content/posts/September-2020.md:52]: That's not 100% correct. It's possible on GitLab and GitHub as well. It just allows selecting namespace where we work with private repositories. (The centos-stream namespace is here only a coincidence or our configuration.)

Comment (sakalosj) [inline content/posts/September-2020.md:52]: @TomasTomecek and @lachmanfrantisek 
will it be ok like this?

```suggestion
- Service supports private namespaces now. They need to be configured in the service config. [packit-service#831](https://github.com/packit/packit-service/pull/831)
```

Comment (TomasTomecek) [reply content/posts/September-2020.md]: my point here was that we are not enabling this in production for github so if anyone in the public reads this, they should know this is our internal thing and we still don't support private repos on github

Comment (TomasTomecek) [reply content/posts/September-2020.md]: I'll open a PR.

**Pull request merged**